README.md

```diff
@@ -178,405 +128,133 @@ After installing the library you will need to require it in your test file (see
+### `setTimeout`
 
-```js
-const protractorHelper = require("protractor-helper");
+That method allows to change the timeout duration of all `protractor-helper` methods called below `setTimeout`.
 
-describe("foo", () => {
-    it("bar", () => {
-        browser.get("https://example.com");
+The `timeoutInMilliseconds` default is 5000 milliseconds
 
-        const myButton = element(by.id("my-button"));
+If is called without pass argument the timeout of the tests is changed to the default timeout.
+[Example](docs/EXAMPLES.md#settimeout)
 
-        protractorHelper.waitForTextToBePresentInElement(myButton, "ENTER", "ENTER text is not present on myButton element", 3000);
+### `getBodyElementFromCurrentBrowserOrBrowserInstance`
 
-        // ...
-    });
-});
-```
+That method returns the body element of the current browser if nothing is passed as argument or the body element of a specific browser instance in case the browser instance is passed as an argument. This second option is useful when working with two browsers interacting with each other, for example.
+ [Example](docs/EXAMPLES.md#getbodyelementfromcurrentbrowserorbrowserinstance)
 
-### Example of usage of `waitForTextNotToBePresentInElement`
+### `openNewBrowserInTheSamePage`
 
-```js
-const protractorHelper = require("protractor-helper");
+That method opens a new browser instance in the same page of the main browser.
 
-describe("foo", () => {
-    it("bar", () => {
-        browser.get("https://example.com");
+> Note: Only works with Protractor version 5.0.0, not higher.
 
-        const myButton = element(by.id("my-button"));
+[Example](docs/EXAMPLES.md#opennewbrowserinthesamepage)
 
-        protractorHelper.waitForTextNotToBePresentInElement(myButton, "ENTER", "ENTER text is still present on myButton element", 3000);
+### `isCurrentUrlDifferentFromBaseUrl`
 
-        // ...
-    });
-});
-```
+That method returns a boolean depending if the current url is different from the base url. No argument needed.
+[Example](docs/EXAMPLES.md#iscurrenturldifferentfrombaseurl)
 
-### Example of usage of `waitForUrlToBeEqualToExpectedUrl`
+### `waitForElementPresence`
 
-```js
-const protractorHelper = require("protractor-helper");
+That method waits for an element to be present in the DOM.
 
-describe("foo", () => {
-    it("bar", () => {
-        browser.get("https://example.com");
+> Note: An element may be present but not displayed. If you want to wait for the element to be displayed use the method `waitForElementVisibility`.
 
-        const goToContactPageButton = element(by.className("contact-button"));
+[Example](docs/EXAMPLES.md#waitforelementpresence)
 
-        goToContactPageButton.click();
+### `waitForElementNotToBePresent`
 
-        protractorHelper.waitForUrlToBeEqualToExpectedUrl("https://example.com/contact", "URL is different from expected", 3000);
+That method is the opposite of the previous one, so, it waits for an element not to be present in the DOM.
```

Comment:
What not saying 'This method', instead of 'That method'?

Reply:
Commit made because this review: ce8ef21